webrtc.html

```diff
@@ -2075,6 +1826,314 @@
   </section>
 
   <section>
+      <h2>RTP Media API</h2>
+
+      <p>The RTP media API lets a web application send and receive <code>MediaStreamTrack</code>s
+      over a peer-to-peer connection. Tracks, when added to a RTCPeerConnection, result in
+      signaling; when this signaling is forwarded to a remote peer, it causes
+      corresponding tracks to be created on the remote side.</p>
+
+      <p>The actual encoding and transmission of <code>MediaStreamTrack</code>s is managed through
+      objects called <code>RTCRtpSender</code>s. Similarly, the reception and decoding of
+      <code>MediaStreamTrack</code>s is managed through objects called <code>RTCRtpReceiver</code>s.</p>
```

Comment:
I think we need more here to explain what a RtpSender / Receiver represents. I'm trying to get at the issue of how many of them do you have. Are they 1:1 with tracks, etc ...